README.md

```diff
@@ -42,35 +42,26 @@ const KERNEL_ACCOUNT_FACTORY_ADDRESS =
 // This is just one exapmle of how to interact with EOAs, feel free to use any other interface
 const owner = PrivateKeySigner.privateKeyToAccountSigner(PRIVATE_KEY);
 
-const validator: ECDSAValidator = new ECDSAValidator(({
-    validatorAddress: "0x180D6465F921C7E0DEA0040107D342c87455fFF5",
-    mode: ValidatorMode.sudo,
-    owner,
-    chain: polygonMumbai,
-    entryPointAddress: "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789"
-}))
-
-// 2. initialize the provider and connect it to the account
-const provider = new ZeroDevProvider({ 
-  chain: polygonMumbai,
-  projectId, // ZeroDev projectId
-  entryPointAddress: "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789" 
-}).connect((rpcClient) =>
-  new KernelSmartContractAccount({
+// 2. Instantiate a Provider Builder
+const providerBuilder = new ProviderBuilder();
+
+// 3. Instantiate a Kernel Provider by passing in the Provider Builder instance
+// Kernel Provider is a preset which initiate the Provider Builder with all the necessary configurations
+// You can can build your own Provider builder with custom configurations if you want
+// without using the kernel provider
+const kernelProvider = new KernelProvider(providerBuilder);
```

Comment:
So the `KernelProvider` is just here to configure the provider builder?  It seems a bit unintuitive that you create an instance of `KernelProvider`, calls `init`, and then never use it again (you go back to using the provider builder).